### PR TITLE
team members are updated according to their user name

### PR DESCRIPTION
**Pull request by tarunnjoshi**
Targets #2429

### Description
Team Members are updated according to their user names.

### Checklist:
- [x] I have performed a self-review of my code.

### CODE REVIEW

**Comment by gautam071** [comment]
@GauravGusain98 Can you review this PR once

File: Modules/EffortTracking/Services/EffortTrackingService.php [hunk 1]
```diff
@@ -191,6 +191,18 @@ public function getEffortForProject($project)
...
+            $range = config('efforttracking.default_start_column_in_effort_sheet') . '2:' . config('efforttracking.default_start_column_in_effort_sheet');
+            $sheet = $sheets->spreadsheet($sheetId)
+                ->range($range)
+                ->get();
+
+            foreach ($sheet as $rows) {
+                if (count($rows) == 0) {
+                    break;
+                }
+                $projectMembersCount++;
```
Comment (GauravGusain98) [inline Modules/EffortTracking/Services/EffortTrackingService.php:203]: According to current logic the teamember will already have the value and this for loop will do an increment over that value. The initial value should be 0 of `projectMembersCount` at line 189. Then we will get the exact count.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

File: Modules/EffortTracking/Services/EffortTrackingService.php (hunk 1 above)
Comment (tarunnjoshi) [reply Modules/EffortTracking/Services/EffortTrackingService.php:203]: Done with the changes.

**Pull request merged**